Testscripts/Linux/fpga-card-validation.sh

```diff
@@ -135,7 +162,7 @@ function validate_cards() {
 # Source utils.sh
 . utils.sh || {
     echo "ERROR: unable to source utils.sh!"
-    echo "TestAborted" > state.txt
```

Comment:
When we have issues during source utils.sh, we can't use function SetTestStateAborted

Reply:
reverted back to what it was